thoth/adviser/cli.py

```diff
@@ -212,8 +212,8 @@ def advise(click_ctx, requirements, requirements_format=None, requirements_locke
             # If we have something to suggest, add it to output field.
             # Do not replicate input to output without any reason.
             if requirements_format == PythonRecommendationOutput.PIPENV:
-                output_requirements = project.pipfile.to_string()
-                output_requirements_locked = project.pipfile_lock.to_string()
+                output_requirements = project.pipfile.to_dict()
```

Comment:
and the dict will be transformed into json on stdout?

Reply:
It will be transformed with all the metadata in the ``print_command_method``.